app/helpers/application_helper.rb

```diff
@@ -93,13 +93,40 @@ def app_version_number
     @app_version_number ||= version
   end
 
-  def division_select_options(include_root: true, include_all: false, public_only: true)
+  def division_select_options(include_root: true, include_all: false, public_only: false)
```

Comment:
this flag was set to false everywhere except the division dropdown on public pages. we may want to bring in the division dropdown based on what we learn from brendan in demo tmrw.